src/Microsoft.ML.ImageAnalytics/ImageLoader.cs

```diff
@@ -196,7 +196,11 @@ protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, b
                                 string path = src.ToString();
                                 if (!string.IsNullOrWhiteSpace(_parent.ImageFolder))
                                     path = Path.Combine(_parent.ImageFolder, path);
-                                dst = new Bitmap(path);
+
+                                using (var stream = new FileStream(path, FileMode.Open, FileAccess.Read))
+                                {
```

Comment:
"You must keep the stream open for the lifetime of the Bitmap. "
https://docs.microsoft.com/en-us/dotnet/api/system.drawing.bitmap.-ctor?view=netframework-4.8#System_Drawing_Bitmap__ctor_System_IO_Stream_

Reply:
I don't it matters if you are not writing to the image. In our case we are not doing that.

Reply:
> It doesn't matters if you are not writing to the image. In our case we are not doing that. The reason they want this is because of `Save` routine we have in Bitmap class.

Interesting. We ought to note this in documentation then, what you think?

Reply:
I have reverted the change now.

---
In reply to: [279979459](https://github.com/dotnet/machinelearning/pull/3625#discussion_r279979459) [](ancestors = 279979459)